.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/**
```

Comment:
This should be redundant afaik.

Reply:
Hmm, I guess I can remove the whole file then.

Reply:
Done.